lib/ecto/repo.ex

```diff
@@ -498,6 +506,35 @@ defmodule Ecto.Repo do
               {:ok, Ecto.Model.t} | {:error, Ecto.Changeset.t}
 
   @doc """
+  Inserts or updates a model or a changeset depending on whether the model
+  was loaded from the database or created by the user. This is especially
```

Comment:
Change to "is persisted or not". I don't like using the word "user" because it's not clear if it's the developer or the end user of the application.

Also :scissors: "especially" :)